src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2117,6 +2117,8 @@ internal enum ErrorCode
         ERR_UnscopedScoped = 9066,
         WRN_DuplicateAnalyzerReference = 9067,
 
+        ERR_readonly_is_not_supported_as_a_parameter_modifier_Did_you_mean_in = 9068,
```

Comment:
This is not the compiler convention. Please use PascalCase and a shorter identifier.

Reply:
Abbreviated identifiers just mean less clarity and more difficulty figuring out what the code is doing. I don't even know how one goes about making the abbreviation. It's it just whacking off parts of the word till it fits some acceptable length?  :-)